### DISCUSSION TITLE
如何自己指定MessageBox.ShowOverlayAsync的id 和toplevelHashCode

### DISCUSSION THREAD

**Discussion by 13572743126** [irihitech/Ursa.Avalonia#512] [Q&A]
在弹出提示框的时候，虽然能够看明白使用Overlay形式时，必须在OverlayDialogManager.GetHost(hostId, toplevelHashCode);函数中获取到当前界面，但是要怎么定义一个列表，将当前Window压入列表？
因为不懂用户如何自己定义id和code,所以没法执行这个函数？
大佬有空了帮我看看

**Comment by rabbitism** [comment]
如果你整个程序只有一个OverlayDialogHost，那么你并不需要指定ID和Hash。

如果你有多个Host，那么ID是自己指定的
https://github.com/irihitech/Ursa.Avalonia/blob/a8891cf9bf36125cee1349c01dfa179227d1772a/demo/Ursa.Demo/Pages/DialogDemo.axaml#L207

toplevel hash就是这个Host所在的Window的HashCode。
